.yarn/plugins/plugin-list.js

```diff
@@ -0,0 +1,138 @@
+// Workaround for https://github.com/microsoft/vscode-vsce/issues/517
+// taken from https://gist.githubusercontent.com/arendjr/415747652a8c79f12b005ce3cfb2f808/raw/a4a227070319e1d4c61e982b74153765d6338694/list-plugin.js
+
+const fs = require('fs');
+
+// Setup: Place this file in `.yarn/plugins/list-plugin.js` and the following
```

Comment:
the path indicated is not the same than the one it is really used list-plugin.js <-> plugin-list.js